.github/CODEOWNERS

```diff
@@ -22,6 +22,15 @@
 /src/plugins/data_view_editor/ @elastic/kibana-data-discovery
 /src/plugins/data_view_field_editor/ @elastic/kibana-data-discovery
 /src/plugins/data_view_management/ @elastic/kibana-data-discovery
+/src/plugins/data/ @elastic/kibana-visualizations @elastic/kibana-data-discovery
+/src/plugins/field_formats/ @elastic/kibana-data-discovery
+/x-pack/plugins/runtime_fields @elastic/kibana-data-discovery
```

Comment:
just to be sure @timductive @mattkime @davismcphee @elastic/kibana-data-discovery owns runtime fields from now on?

Reply:
Its only used by `index_management` so I think it should be owned by the stack management team

Reply:
I've changed it to @elastic/platform-deployment-management team; @mattkime is that the one you had in mind?

Reply:
just to bring this to everyone's awareness: @elastic/platform-deployment-management is paused at the moment. cc @ranjanik200 @alisonelizabeth

Reply:
> Its only used by index_management 

Is this a recent change? I thought it was being used elsewhere.

As Yulia mentioned, the Deployment Management team is currently not staffed so I'd prefer the team alias does not acquire more plugins as code owner (although I understand the initial reasoning if it is only being used by Index Management).

Reply:
>Is this a recent change? I thought it was being used elsewhere.

Its not a recent change. It was never used in its second location. Maybe this should be reexamined.